llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -7420,11 +7414,6 @@ foreach fvti = AllFloatVectors in {
                (fvti.Vector $rs1),
```

Comment:
Why do we still need this pattern?

Reply:
We don't, I originally removed it but added it back in as per @preames suggestion https://github.com/llvm/llvm-project/pull/71501#discussion_r1392997610 (looks like GitHub isn't showing the comment inline anymore though 😬)